# HG changeset patch
# User Philip Nienhuis <prnienhuis@users.sf.net>
# Date 1485703473 -3600
#      Sun Jan 29 16:24:33 2017 +0100
# Node ID 19e9589744102292d9f6767da23f227da149dfaa
# Parent  9241a0fa78732d006a22299f9aaef1c9b775b368
fminsearch.m: fix texinfo header (bug #50178)

diff --git a/scripts/optimization/fminsearch.m b/scripts/optimization/fminsearch.m
--- a/scripts/optimization/fminsearch.m
+++ b/scripts/optimization/fminsearch.m
@@ -41,17 +41,17 @@
 ## and higher arguments.  To pass function arguments while using the default
 ## @var{options} values, use @code{[]} for @var{options}.
 ##
 ## On exit, the function returns @var{x}, the minimum point, and @var{fval},
 ## the function value at the minimum.
 ##
 ## The third return value @var{exitflag} is
 ##
-## @table
+## @table @asis
 ## @item 1
 ## if the algorithm converged
 ## (size of the simplex is smaller than @code{@var{options}.TolX} @strong{AND}
 ## the step in the function value between iterations is smaller than
 ## @code{@var{options}.TolFun}).
 ##
 ## @item 0
 ## if the maximum number of iterations or the maximum number of function
